main/src/ca/uwaterloo/flix/language/phase/Weeder2.scala

```diff
@@ -489,7 +489,7 @@ object Weeder2 {
       ) {
         (doc, ident, tparams, fields) =>
           // Ensure that each name is unique
-          val errors = getDuplicates(fields, (f: StructField) => f.name.name).map {
+          val errors = SeqOps.getDuplicates(fields, (f: StructField) => f.name.name).map {
```

Comment:
Should be separate PR, but I guess we're in crazy town.

Reply:
I can pull it out into another PR no worries